src/services/ros/index.ts

```diff
@@ -29,10 +29,26 @@ export interface IUser {
   isAdmin: boolean;
   accounts: IAccount[];
   metadata: IUserMetadataRow[];
+  status: UserStatus;
 }
 
 export type User = IUser & Realm.Object;
 
+export enum UserStatus {
+  active = 'active',
+  blacklisted = 'blacklisted',
+}
+
+export function humanizeUserStatus(userStatus: UserStatus) {
```

Comment:
Would probably move this to the component, as it is concerned with how data is displayed and not used by other components yet.